client/components/app.jsx

```diff
@@ -1,8 +1,23 @@
 import React from 'react';
-
+import Header from './header';
+import GradeTable from './grade-table';
 class App extends React.Component {
+  constructor(prop) {
+    super(prop);
+    this.state = { grades: [] };
+  }
+
+  componentDidMount() {
+    fetch('/api/grades', { method: 'GET' }).then(res => res.json()).then(data => this.setState({ grades: data }));
```

Comment:
The default method for `fetch()` is get, so you can leave out the second argument.